app/javascript/src/Users/permissions.jsx

```diff
@@ -228,7 +228,8 @@ export const Form = { // eslint-disable-line no-unused-vars
       settings: 'Settings',
       partners: 'Developer Accounts -- Applications',
       monitoring: 'Analytics',
-      plans: 'Integration & Application Plans'
+      plans: 'Integration & Application Plans',
+      policy_registry: 'Policy Registry'
```

Comment:
Use cammelCase instead

Reply:
I do not think so because those are keys coming from Rails.